docs/guides/shuffle-sharding.md

```diff
@@ -154,6 +155,20 @@ Cortex ruler can run in three modes:
 
 Note that when using sharding strategy, each rule group is evaluated by single ruler only, there is no replication.
 
+### Compactor shuffle sharding
+
+Cortex compactor can run in three modes:
+
+1. **No sharding at all.** This is the most basic mode of the compactor. It is activated by using `-compactor.sharding-enabled=false` (default). In this mode every compactor will run every compaction.
+2. **Default sharding**, activated by using `-compactor.sharding-enabled=true` and `-compactor.sharding-strategy=default` (default). In this mode compactors register themselves into the ring. Each compactor will then select and evaluate only those users that it "owns".
```

Comment:
I think we should be more clear on the difference between "default sharding" and "shuffle sharding". I don't feel `Each compactor will then select and evaluate only those users that it "owns".` and `...but compactions groups for each tenant can only be evaluated on limited number of compactors` clearly state the difference between the two different sharding strategy. I would expect the doc have words like "single compactor" and "multiple compactors".